sycl/include/CL/sycl/detail/properties_traits.def

```diff
@@ -5,6 +5,7 @@ __SYCL_PARAM_TRAITS_SPEC(sycl::property::image::use_host_ptr)
 __SYCL_PARAM_TRAITS_SPEC(sycl::property::image::use_mutex)
 __SYCL_PARAM_TRAITS_SPEC(sycl::property::image::context_bound)
 __SYCL_PARAM_TRAITS_SPEC(sycl::ext::oneapi::property::buffer::use_pinned_host_memory)
+__SYCL_PARAM_TRAITS_SPEC(sycl::INTEL::property::kernel::gpu_cache_config)
```

Comment:
I think all these `SYCL::INTEL` namespaces should be `sycl::ext::intel` instead.  For example, look at the line immediately above, which has `sycl::ext::oneapi`.